services/postprocessing/pkg/config/config.go

```diff
@@ -23,15 +23,15 @@ type Config struct {
 // Postprocessing definces the config options for the postprocessing service.
 type Postprocessing struct {
 	Events          Events        `yaml:"events"`
-	Virusscan       bool          `yaml:"virusscan" env:"POSTPROCESSING_VIRUSSCAN" desc:"should the system do a virusscan? Needs antivirus service"`
-	Delayprocessing time.Duration `yaml:"delayprocessing" env:"POSTPROCESSING_DELAY" desc:"the sytem sleeps for this time while postprocessing"`
+	Virusscan       bool          `yaml:"virusscan" env:"POSTPROCESSING_VIRUSSCAN" desc:"Needs as prerequisite the antivirus service to be enabled and configured. should the system do a virusscan?"`
+	Delayprocessing time.Duration `yaml:"delayprocessing" env:"POSTPROCESSING_DELAY" desc:"A delay in the given duration during the sytem will sleep while postprocessing. The duration can be set as number followed by a unit identifyer like s, m or h."`
```

Comment:
```suggestion
	Delayprocessing time.Duration `yaml:"delayprocessing" env:"POSTPROCESSING_DELAY" desc:"A delay in the given duration during the system will sleep while postprocessing. The duration can be set as number followed by a unit identifier like s, m or h."`
```

Reply:
"A delay in the given duration during the system will sleep while postprocessing" - I don't understand this. If I know what it actually does, then I could suggest how to edit the words ;)

Reply:
You are completely right 👍 
I added this change to easily start reviewing and to add clarity as I have written in my PR description.

Reply:
You are completely right 👍 
I added this change to easily start reviewing and to add clarity as I have written in my PR description.

Reply:
Yeah I wasn't sure how to phrase this best. I'll try to describe:

During postprocessing the file will be in `425` state (processing) during which it cannot be downloaded. To be able to test this one can set to e.g. `30s`. Ocis will then sleep during postprocessing for 30 seconds. That means the minimum amount of time this file is in processing is 30s.

To be clear: This envvar enables developers to test their new features against file that are in processing. It is not meant to be set on a productive enviroment. (Unless as admin you want to annoy your users.